assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,23 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
```

Comment:
awesome! I'm glad you decided to take this assignment on. I feel it should be mandatory for everyone because of how useful and important it is. 

Great job Davina.